krystle_talens-fatima.js

```diff
@@ -0,0 +1,124 @@
+//Calculate Cube
+function calculateCube(num) {
+    console.log(num*num*num)
+  }
+  
+  calculateCube(5)
+  
+//Is a Vowel?
+  var vowels = [ "a", "e", "i", "o", "u","y"] 
+  const isAVowel = (letter) => {
+    if (vowels.includes(letter)) {
+      return true
+    } else {
+      return false
+    }
+  }  
+  console.log(isAVowel("a"));
+
+//Get Two Lengths
+    const getTwoLengths = (string1,string2) => {
+    var x = [];
+    x.push(string1.length);
+    x.push(string2.length);
+    return x
+  }
+  console.log(getTwoLengths("Hank", "Hippopopalous"));
+
+//Get Multiple Lengths
+  function getMultipleLengths(array) {
+    y=[]
+    for (i=0; i<array.length;i++){
+        y.push(array[i].length)
+        }   
+    return y
```

Comment:
careful with your indentation in line 33! your closing bracket should be as far left as the line where its counterpart is